### PR TITLE
Add AWS Glue Crawler permissions

### PR DESCRIPTION
**Pull request by ichekaldin**
This allows AWS Glue crawler integration tests to be executed:
https://github.com/ansible-collections/community.aws/pull/546

### CODE REVIEW

File: aws/policy/application-services.yaml [hunk 1]
```diff
@@ -107,12 +107,15 @@ Statement:
       - kinesis:RemoveTagsFromStream
       - kinesis:StartStreamEncryption
       - kinesis:StopStreamEncryption
+      - glue:DeleteCrawler
       - glue:DeleteJob
+      - glue:GetCrawler
```
Comment (jillr) [inline aws/policy/application-services.yaml:112]: We also need glue:GetCrawlers, for use by the terminator class.  This needs to operate globally, so it should go under `  - Sid: AllowGlobalUnrestrictedResourceActionsWhichIncurNoFees`

**Review by jillr — CHANGES_REQUESTED** [review]
Thanks @ichekaldin. The permissions and terminator that are here look ok but I'm not able to test them fully with your other PR.  I'll leave some comments on 546.

File: aws/policy/application-services.yaml (hunk 1 above)
Comment (ichekaldin) [reply aws/policy/application-services.yaml:112]: @jillr Thank you for reviewing this - that's an interesting point.

I see that there are no other Glue-related permissions under `Sid: AllowGlobalUnrestrictedResourceActionsWhichIncurNoFees`.

If they are in fact needed for terminator class, should I add permissions for Glue crawlers, jobs, and connections:

```
- glue:GetConnections
- glue:GetCrawlers
- glue:GetJobs
```

What do you think?

Comment (jillr) [reply aws/policy/application-services.yaml:112]: @ichekaldin Yes please, thanks!

Comment (ichekaldin) [reply aws/policy/application-services.yaml:115]: @jillr Done

**Review by jillr — APPROVED** [review]
Thanks @ichekaldin  I'm still getting boto errors running the test suite locally but these policies look good.  I'll get these merged and deployed so we can debug in CI.

**Pull request merged**